streamer/pulp/streamer/server.py

```diff
@@ -278,28 +316,20 @@ def __enter__(self):
         """
         return self
 
-    def __exit__(self, exc_type, exc_val, exc_tb):
+    def __exit__(self, *unused):
```

Comment:
This change seems a little risky, in case it got called with kwargs. I don't know if the language makes any guarantees about which way this will get called.